docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
 .. _release_and_maintenance:
 
-Release and maintenance
-=======================
+************************
+Releases and maintenance
+************************
 
-This section describes the Ansible and ``ansible-core`` releases.
-Ansible is the package that most users install. ``ansible-core`` is
-primarily for developers.
+This section describes release cycles, rules, and maintenance schedules for both Ansible community projects: the Ansible community package and ``ansible-core``. The two projects have different versioning systems, maintenance structures, contents, and workflows.
+
+====================================================  =================================================
+Ansible community package                             ansible-core
+====================================================  =================================================
+each major release is a new major version (4.0.0)     continues "classic Ansible" versioning (2.11)
+follows semantic versioning rules                     does not use semantic versioning
+maintains only one version at a time                  maintains latest version plus two older versions
+includes language, runtime, and selected Collections  includes only the language and runtime
+developed and maintained in Collection repositories   developed and maintained in ansible/ansible repo
```

Comment:
@acozine 
I think you should use sentence-style capitalization in the table text.

Reply:
thanks @lmalivert - addressed in cca19eef3206585b6f806ace09cba21a993cab28

Reply:
@acozine LGTM 👍🏽 Thanks!